app/controllers/workflows_controller.rb

```diff
@@ -128,6 +128,12 @@ def destroy
   def submit
     set_workflow
 
+    # We want to allow the user to resubmit a job that has been run or failed. This will clear old jobs from
```

Comment:
You mean `clear`?

I don't remember why `clear` works and `destroy` doesn't. I'll figure it out again and I may need to put an issue in `osc-machete` or `osc_machete_rails`

```
irb(main):009:0> Workflow.last.jobs
  Workflow Load (5.1ms)  SELECT  "workflows".* FROM "workflows"   ORDER BY "workflows"."id" DESC LIMIT 1
  Job Load (3.7ms)  SELECT "jobs".* FROM "jobs"  WHERE "jobs"."workflow_id" = ?  [["workflow_id", 82]]
=> #<ActiveRecord::Associations::CollectionProxy [#<Job id: 27, workflow_id: 82, status: "Q", job_cache: {"sc
ript"=>"/users/appl/bmcmichael/ondemand/dev/myjobs/data/projects/default/66/pbs_script.sh", "pbsid"=>"9327595
.oak-batch.osc.edu", "host"=>"oakley"}, created_at: "2017-05-19 15:04:21", updated_at: "2017-05-19 15:04:21">
]>
irb(main):010:0> Workflow.last.jobs.destroy
  Workflow Load (3.0ms)  SELECT  "workflows".* FROM "workflows"   ORDER BY "workflows"."id" DESC LIMIT 1
=> []
irb(main):011:0> Workflow.last.submitted?  
  Workflow Load (1.7ms)  SELECT  "workflows".* FROM "workflows"   ORDER BY "workflows"."id" DESC LIMIT 1
  Job Load (1.7ms)  SELECT "jobs".* FROM "jobs"  WHERE "jobs"."workflow_id" = ?  [["workflow_id", 82]]
=> true
irb(main):012:0> Workflow.last.jobs.clear  
  Workflow Load (2.0ms)  SELECT  "workflows".* FROM "workflows"   ORDER BY "workflows"."id" DESC LIMIT 1
  SQL (8.5ms)  DELETE FROM "jobs" WHERE "jobs"."workflow_id" = ?  [["workflow_id", 82]]
=> #<ActiveRecord::Associations::CollectionProxy []>
irb(main):013:0> Workflow.last.submitted?
  Workflow Load (2.0ms)  SELECT  "workflows".* FROM "workflows"   ORDER BY "workflows"."id" DESC LIMIT 1
  Job Load (1.7ms)  SELECT "jobs".* FROM "jobs"  WHERE "jobs"."workflow_id" = ?  [["workflow_id", 82]]
=> false
```